### PR TITLE
ci: added rules for update version in Cargo.toml anmd pyproject.toml files

### PR DESCRIPTION
**Pull request by khalezin**
ci: added rules for update version in Cargo.toml and pyproject.toml files

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -34,10 +34,23 @@ const config = {
...
+          "toml set --toml-path substreams/evm-ambient/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/evm-uniswap-v2/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/evm-uniswap-v3/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/substreams-helper/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path token-analyzer/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path tycho-client/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path tycho-client-py/pyproject.toml project.version ${nextRelease.version}",
+          "toml set --toml-path tycho-core/Cargo.toml project.version ${nextRelease.version}",
+          "toml set --toml-path tycho-indexer/Cargo.toml project.version ${nextRelease.version}",
+          "toml set --toml-path tycho-storage/Cargo.toml project.version ${nextRelease.version}",
```
Comment (khalezin) [inline release.config.js:48]: @kayibal This is the list of Cargo.toml and pyproject.toml files.

File: release.config.js [hunk 2]
```diff
@@ -34,10 +34,23 @@ const config = {
...
         verifyReleaseCmd:
           'echo "VERIFY_RELEASE_VERSION=${nextRelease.version}" >> $GITHUB_OUTPUT',
+        publishCmd:
+          'echo "NEXT_RELEASE_VERSION=${nextRelease.version}" >> $GITHUB_OUTPUT',
+        prepareCmd: [
+          "toml set --toml-path substreams/common/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/evm-ambient/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/evm-uniswap-v2/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/evm-uniswap-v3/Cargo.toml package.version ${nextRelease.version}",
+          "toml set --toml-path substreams/substreams-helper/Cargo.toml package.version ${nextRelease.version}",
```
Comment (kayibal) [inline release.config.js]: Hmm can we exclude those for now - we plan to move these to a separate repo.

File: release.config.js [hunk 3]
```diff
@@ -34,10 +34,23 @@ const config = {
...
       "@semantic-release/exec",
       {
-        publishCmd:
-          'echo "NEXT_RELEASE_VERSION=${nextRelease.version}" >> $GITHUB_OUTPUT',
         verifyReleaseCmd:
           'echo "VERIFY_RELEASE_VERSION=${nextRelease.version}" >> $GITHUB_OUTPUT',
+        publishCmd:
+          'echo "NEXT_RELEASE_VERSION=${nextRelease.version}" >> $GITHUB_OUTPUT',
+        prepareCmd: [
+          "toml set --toml-path substreams/common/Cargo.toml package.version ${nextRelease.version}",
```
Comment (kayibal) [inline release.config.js:42]: This one can stay though.

File: release.config.js (hunk 2 above)
Comment (khalezin) [reply release.config.js]: Ok, no problem

**Pull request merged**